forest/daemon/src/daemon.rs

```
@@ -337,6 +339,7 @@ pub(super) async fn start(config: Config, detached: bool) -> Db {
                 FOREST_VERSION_STRING.as_str(),
             )
             .await;
+            Ok(())
```

Why ignore the error message here?

---

Fixed